client/src/components/qa/AddAnswer.jsx

```diff
@@ -46,13 +46,15 @@ function AddAnswer({
             <h1 className="qa-add-title">Submit Your Answer </h1>
             <h2>{name}: {body}</h2>
             <form onSubmit={handleAddAnswer}>
+              <span>(<span style={{ color: 'red' }}>*</span>) Mandatory</span>
               <h3>Your Answer <span style={{ color: 'red' }}>*</span></h3>
               <textarea
                 className="qa-add-input"
                 rows="5"
                 cols="40"
                 maxLength="1000"
                 onChange={(e) => { setAnswerInput(e.target.value); }}
+                required
```

Comment:
Great that you added it here too, it's a nice touch to have all the mandatory fields required!